main.go

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright © 2022 four-cryptics
+*/
+
+package main
```

Comment:
Let's follow the convention for Go project structure.

All the main files should be in the /cmd/ folder, those can be defined as package main.

From what I see, this file can be put into /cmd/ and all the contents of `root.go` can be copied here while deleting `root.go`.